pkg/sources/source_manager_test.go

```diff
@@ -207,7 +207,7 @@ func (c *unitChunker) ChunkUnit(ctx context.Context, unit SourceUnit, rep ChunkR
 			continue
 		}
 		if step.err != "" {
-			if err := rep.ChunkErr(ctx, fmt.Errorf(step.err)); err != nil {
+			if err := rep.ChunkErr(ctx, fmt.Errorf("%s", step.err)); err != nil {
```

Comment:
%w for error wrapping?

Reply:
This is in a test and `step.err` is a `string` so it's just converting the `string` into an `error` for testing.